apps/dash-label-properties/app.py

```diff
@@ -0,0 +1,357 @@
+                            html.Img(
+                                src=app.get_asset_url("dash-logo-new.png"),
+                                height="30px",
+                            ),
+                            href="https://plotly.com/dash/",
+                        )
+                    ),
+                    dbc.Col(dbc.NavbarBrand("Object Properties App")),
+                    modal,
+                ],
+                align="center",
+            ),
+            dbc.Row(
+                dbc.Col(
+                    [
+                        dbc.NavbarToggler(id="navbar-toggler"),
+                        dbc.Collapse(
+                            dbc.Nav(
+                                [dbc.NavItem(button_howto), dbc.NavItem(button_gh)],
+                                className="ml-auto",
+                                navbar=True,
+                            ),
+                            id="navbar-collapse",
+                            navbar=True,
+                        ),
+                    ]
+                ),
+                align="center",
+            ),
+        ],
+        fluid=True,
+    ),
+    color="dark",
+    dark=True,
+)
+
+# Define Cards
+image_card = dbc.Card(
+    [
+        dbc.CardHeader(html.H2("Explore object properties")),
+        dbc.CardBody(
+            dbc.Row(
+                dbc.Col(
+                    dcc.Graph(
+                        id="graph", figure=image_with_contour(img, labels, mode=None),
+                    )
+                )
+            )
+        ),
+    ]
+)
+
+table_card = dbc.Card(
+    [
+        dbc.CardHeader(html.H2("Data Table")),
+        dbc.CardBody(
+            dbc.Row(
+                dbc.Col(
+                    [
+                        dash_table.DataTable(
+                            id="table-line",
+                            columns=columns,
+                            data=table.to_dict("records"),
+                            filter_action="native",
+                            row_deletable=True,
+                            style_table={"overflowY": "scroll"},
+                            fixed_rows={"headers": False, "data": 0},
+                            style_cell={"width": "85px"},
+                        ),
+                        dcc.Store(id="cache", data=labels),
+                        html.Div(id="row", hidden=True, children=None),
+                    ]
+                )
+            )
+        ),
+    ]
+)
+
+app.layout = html.Div(
+    [
+        header,
+        dbc.Container(
+            [dbc.Row([dbc.Col(image_card, md=5), dbc.Col(table_card, md=7)])],
+            fluid=True,
+        ),
+    ]
+)
+
+
+@app.callback(
+    Output("table-line", "style_data_conditional"), [Input("graph", "hoverData")]
+)
+def higlight_row(string):
+    """
+    When hovering hover label, highlight corresponding row in table,
+    using label column.
+    """
+    if not dash.callback_context.triggered:
```

Comment:
I have added this callback check (and a similar one in line 319) to "catch" None-type errors right when the app starts. In the original app, there was a `suppress_callback_exceptions` that I have removed and that I believe was intended to take care of this. Maybe there is a better way of dealing with these issues

Reply:
I'm not sure what the original intention is (remember that this was maybe the first Dash app I ever wrote so expect the code to be really bad ;-)). But we now have the possibility to pass `prevent_initial_call=True` in the arguments of `app.callback` which seems like the way to go here.

Reply:
oh nice, I'll use this instead, thanks! Haha, I wasn't sure if it was something that worked in an older version or where this was coming from. I think the errors only showed up in the console or during debug mode so they didn't interfere with the functionality.